systemtest/src/main/resources/StrimziUpgradeST.json

```diff
@@ -157,8 +157,8 @@
       "userOperator": "strimzi/operator:0.16.2"
     },
     "imagesAfterKafkaUpdate": {
-      "zookeeper": "strimzi/kafka:0.16.2-kafka-2.4.0",
-      "kafka": "strimzi/kafka:0.16.2-kafka-2.4.0",
+      "zookeeper": "strimzi/kafka:0.16.2-kafka-2.4.1",
+      "kafka": "strimzi/kafka:0.16.2-kafka-2.4.1",
```

Comment:
For upgrade from `0.15.0` to `0.16.2` you cannot change kafka version to `2.4.1` because there are no kafka 2.4.1 for it. Keep `2.4.0` here.